src/dummy_audio.cpp

```diff
@@ -0,0 +1,40 @@
+#include "crepe/Sound.h"
+#include "crepe/util/log.h"
+
+#include <chrono>
+#include <thread>
+
+using namespace crepe;
+using namespace std;
```

Comment:
is er een code standaard voor using namespace std of std::cout (voorbeeld)

Reply:
Er staat in contributing.md dat `using namespace` alleen mag in bronbestanden en niet in headers